src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -26,7 +26,7 @@ namespace Tizen.NUI.BaseComponents
     /// <summary>
     /// WebView
```

Comment:
like "WebView allows presenting content with embedded web browser, both local files and remote websites."

Reply:
Done.